package/kernel/linux/modules/other.mk

```diff
@@ -64,6 +64,26 @@ endef
 
 $(eval $(call KernelPackage,bluetooth))
 
+define KernelPackage/at803x
+  SUBMENU:=$(OTHER_MENU)
+  TITLE:=PHY at803x Kernel Module support
+  DEPENDS:=+kmod-i2c-core
+  KCONFIG:= \
+	CONFIG_SFP=m \
+	CONFIG_MDIO_I2C=m
+  FILES:= \
+	$(LINUX_DIR)/drivers/net/phy/at803x.ko \
+	$(LINUX_DIR)/drivers/net/phy/sfp.ko \
+	$(LINUX_DIR)/drivers/net/phy/mdio-i2c.ko
```

Comment:
I think it is reasonable to separate these kmods into different packages, then use the dependencies to make it clear

Reply:
I added a SFP package and set at803x as a build-in for now.